sympy/matrices/sparse.py

```diff
@@ -252,6 +252,13 @@ def _eval_inverse(self, **kwargs):
                         iszerofunc=kwargs.get('iszerofunc', _iszero),
                         try_block_diag=kwargs.get('try_block_diag', False))
 
+    def reshape(self, r, c):
```

Comment:
It would be better to add this in a superclass rather than here. The `SparseMatrix` class is kind of pointless now because all matrices are internally sparse.